File: app/src/main/java/com/katkov/lolachievements/application/base/ProgressVeiw.java
package com.katkov.lolachievements.application.base;

import com.arellomobile.mvp.viewstate.strategy.AddToEndSingleStrategy;
import com.arellomobile.mvp.viewstate.strategy.StateStrategyType;

public interface ProgressVeiw {

    @StateStrategyType(AddToEndSingleStrategy.class)
    void setProgressEnable(boolean isEnable);
}

File: app/src/main/java/com/katkov/lolachievements/application/base/ProgressVeiw.kt
package com.katkov.lolachievements.application.base

import com.arellomobile.mvp.viewstate.strategy.AddToEndSingleStrategy
import com.arellomobile.mvp.viewstate.strategy.StateStrategyType

interface ProgressVeiw {

    @StateStrategyType(AddToEndSingleStrategy::class)
    fun setProgressEnable(isEnable: Boolean)
}
